news/2018-09-19-the-qat-gazette-september-2018.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "The QAT Gazette: September 2018"
+date: 2018-10-19 16:00:00 +0000
+---
+
+Huge changes to Beatmap Nominator applications, updates on improvements to Modding Discussions, and a brand new blog contributor? What will they come up with next? This month’s issue of the QAT Gazette has all the answers!
```

Comment:
hardly clear that "new blog contributor" just means a new QAT member, and I don't think it's so fitting to ask these "find out in the post!" questions
```suggestion
This month's issue of the QAT Gazette covers huge changes to Beatmap Nominator applications, updates on improvements to Modding Discussions, and changes within the team. Read on to learn more!
```
probably could be re-worded but that is my idea

Reply:
I mean a new person writing on this blog (which is Kibbleru), not just that he's a new QAT since this gets covered in every issue anyway. If I specify that it's a new "QAT Gazette contributor", then I'm using identical phrasing twice in quick succession which sounds worse imo.

Also I literally start every single gazette article with these sentences so /shrug

Reply:
Discussed in discord and removed unnecessary sensationalist commentary.